queries.sql

```diff
@@ -1,34 +1,66 @@
 /*Queries that provide answers to the questions from all projects.*/
```

Comment:
According to the project requirements, the following query should be present. Please add it.

<img width="531" alt="Screen Shot 2022-08-26 at 8 52 05" src="https://user-images.githubusercontent.com/82426682/186919093-f2c98b91-31df-42a2-868b-192972b5e771.png">